build.sbt

```diff
@@ -1139,4 +1139,4 @@ lazy val root = (project in file("."))
   )
 
 addCommandAlias("assemblySamples", ";flinkManagementSample/assembly;standaloneSample/assembly;generic/assembly")
-addCommandAlias("assemblyEngines", ";flinkProcessManager/assembly;engineStandalone/assembly")
+addCommandAlias("assemblyEngines", ";flinkDeploymentManager/assembly;engineStandalone/assembly")
```

Comment:
IMO we should also rename task to "assemblyDeploymentManagers" or "assemblyDeployers". Also I think that we should scan for usage of word "engine" in such context. I don't mind that is used in package name or module name but if was used as an alias to DeploymentManager - it should be changed.